src/odb/src/db/dbModule.cpp

```diff
@@ -559,11 +560,15 @@ dbModule* dbModule::makeUniqueDbModule(const char* cell_name,
   std::string module_name = orig_cell_name + '_' + std::string(inst_name);
   do {
     std::string full_name = module_name;
+
+    name_id_map_mutex.lock();
```

Comment:
Its better to use std::unique_lock than manual lock and unlock calls (eg exception safety).